.gitattributes

```diff
@@ -9,3 +9,4 @@
 *.gen.json linguist-generated=true
 *_pb2.py linguist-generated=true
 go.sum merge=union
+vendor/**  linguist-vendored
```

Comment:
I don't particularly care, but this PR could be simplified to adding `boringssl-fips/** linguist-vendored` instead.

Reply:
That was my first idea but the .gitattributes is vendored from https://github.com/istio/common-files itself, so while it isn't impossible to have ztunnel-specific things in it, its a bit nicer to have it generic